atomic-examples/quickstart/quickstart/4_basic_chatbot_different_providers.py

```diff
@@ -46,7 +53,7 @@ def setup_client(provider):
 
 
 # Prompt the user to choose a provider
-provider = console.input("[bold yellow]Choose a provider (openai/groq/ollama): [/bold yellow]").lower()
+provider = console.input("[bold yellow]Choose a provider (1. openai / 2. claude / 3. groq / 4. ollama): [/bold yellow]").lower()
```

Comment:
Let's make this
`provider = console.input("[bold yellow]Choose a provider (\[1]. openai / \[2]. anthropic / \[3]. groq / \[4]. ollama): [/bold yellow]").lower()`

This makes it a little bit more clear on first sight that you can make a choice using the numbers.

Reply:
Let's use all the colors!!! :)

![image](https://github.com/user-attachments/assets/4dc51c94-30fe-4ad1-8b92-3086b8e6057f)

Reply:
It was a joke. This is how It looks now:
![image](https://github.com/user-attachments/assets/dfd8a2c6-93a0-45fc-ac95-f2c2182b48ed)

Reply:
Hah okay thought I misread the docs!